web/models.py

```
@@ -25,6 +27,91 @@
+    """Model to store statistics for live counters."""
+
+    COUNTER_TYPES = [
+        ("active_users", "Active Users"),
+        ("enrollments_today", "Enrollments Today"),
+        ("courses_completed", "Courses Completed"),
+        ("quizzes_completed", "Quizzes Completed"),
+        ("total_learners", "Total Learners"),
+        ("total_enrollments", "Total Enrollments"),
+        ("total_courses_completed", "Total Courses Completed"),
+        ("total_quizzes_taken", "Total Quizzes Taken"),
+    ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)